js/test/Exchange/test.fetchLeverageTiers.js

```diff
@@ -12,6 +12,13 @@ module.exports = async (exchange, symbol) => {
     //     ],
     // };
     if (exchange.has[method]) {
+
+        const market = exchange.market (symbol);
+        if (market.spot) {
+            console.log (method + '() is not supported for spot market symbol');
```

Comment:
@ttodua "for spot market symbol" seems a bit strange, did you mean "for spot market" + symbol or smth like that?

Reply:
Maybe it should just be "for spot markets"

Reply:
I've just copied that line from https://github.com/ccxt/ccxt/blob/master/js/test/Exchange/test.fetchFundingRateHistory.js#L25

and yes, you are right, we can amend that. (edit: i've pushed changes for both files, please approve whenever you can).

Reply:
@ttodua nitpick sorry, market should be `markets`